submissions/JuliaNovak/TJSW-pre-oop/index.js

```diff
@@ -0,0 +1,55 @@
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+   species: "dog",
+   name: "Toby",
+   gender: "male",
+   legs: 4,
+   hands: 0,
+   saying: "woof-woof!",
+};
+
+const cat = {
+   species: "cat",
+   name: "Lily",
+   gender: "female",
+   legs: 4,
+   hands: 0,
+   saying: "meow!",
+};
+
+const woman = {
+   species: "human",
+   name: "Linda",
+   gender: "female",
+   legs: 2,
+   hands: 2,
+   saying: "What a pretty kitten!",
+};
+
+const man = {
+   species: "human",
+   name: "Jim",
+   gender: "male",
+   legs: 2,
+   hands: 2,
+   saying: "Heyyo!",
+};
+
+const catWoman = {
+   species: "cat-woman",
+   name: "Ro",
+   gender: "female",
+   legs: 2,
+   hands: 2,
+   saying: cat.saying,
```

Comment:
Task requires your catWoman's saying to always say what cat says. If your cat saying changes, your catwoman's doesn't. It only stores the initial `cat.saying` value.

Reply:
fixed